lib/whitehall_importer/integrity_checker/change_history_check.rb

```diff
@@ -0,0 +1,44 @@
+module WhitehallImporter
+  class IntegrityChecker::ChangeHistoryCheck
+    attr_reader :edition, :proposed_change_history, :publishing_api_change_history
+
+    def initialize(edition, proposed_change_history, publishing_api_change_history)
+      @edition = edition
+      @proposed_change_history = proposed_change_history
+      @publishing_api_change_history = publishing_api_change_history
+    end
+
+    def match?
+      return false unless history_length_matches?
+
+      if edition.draft?
```

Comment:
This will only catch editions in the status of draft whereas we'd want it to cover statuses like submitted for review etc. I'd suggest switching to checking the item isn't live. 

I also think we don't actually need to pass in an edition (I think we only use one for this question) so it's probably simpler that we just pass in a boolean of whether the item is live or not and then can simplify tests a bit (we'd want to switch to a keyword arg on initialise for it)

Reply:
@Rosa-Fox I think from Kevin's comment here he's expecting `live_edition` to be a keyword arg, so `def initialize(live_edition:...`

Reply:
👍